packages/jupytercad-extension/src/mainview.tsx

```diff
@@ -39,9 +39,6 @@ THREE.BufferGeometry.prototype.computeBoundsTree = computeBoundsTree;
 THREE.BufferGeometry.prototype.disposeBoundsTree = disposeBoundsTree;
 THREE.Mesh.prototype.raycast = acceleratedRaycast;
 
-const DARK_BG_COLOR = 'linear-gradient(rgb(0, 0, 42), rgb(82, 87, 110))';
-const LIGHT_BG_COLOR = 'radial-gradient(#efeded, #8f9091)';
-
 const DEFAULT_MESH_COLOR = new THREE.Color('#434442');
 const SELECTED_MESH_COLOR = new THREE.Color('#AB5118');
```

Comment:
Could we make the mesh color a little bit brighter?

Reply:
Yes I'm working on it 👍🏽